docs/fundamentals/sync-modes.md

```diff
@@ -28,7 +28,7 @@ To summarize, snap sync progresses in the following sequence:
 - download block bodies and receipts. In parallel, download raw state data and build state trie
 - heal state trie to account for newly arriving data
 
-**Note** Snap sync is the default behaviour, so if the `--syncmode` value is not passed to Geth at startup, Geth will use snap sync. A node that is started using `snap` will switch to block-by-block sync once it has caught up to the head of the chain.
+<Note>Snap sync is the default behaviour, so if the `--syncmode` value is not passed to Geth at startup, Geth will use snap sync. A node that is started using `snap` will switch to block-by-block sync once it has caught up to the head of the chain.</Note>
```

Comment:
What's the purpose of changing this? Looks good as it is, to me: 

![Screenshot 2023-06-14 at 14-20-48 Sync modes go-ethereum](https://github.com/ethereum/go-ethereum/assets/142290/0cba0424-88be-4548-a51b-d4758db96d84)

Reply:
I was going by the [README on adding a note](https://github.com/ethereum/go-ethereum/tree/website#adding-notes-to-a-doc), and following other notes in the documentation e.g.:

<img width="797" alt="Screenshot 2023-06-14 at 09 09 41" src="https://github.com/ethereum/go-ethereum/assets/3271191/a51ab339-5836-4a5a-93b2-17ca083977d4">

which in my opinion looks for in uniform.

I can revert the commit on this PR or close and open a new PR if you don't see it necessary.
The indentation is definitely a fix IMO.

Reply:
Ah, ok, that's fine then!